config/src/config/network_config.rs

```diff
@@ -151,7 +149,8 @@ impl NetworkConfig {
             if !network_role.is_validator() && self.enable_remote_authentication {
                 ensure!(
                     self.peer_id == peer_id,
-                    "For full-nodes that use remote authentication, the peer_id must be derived from the identity key.",
+                    "For full-nodes that use remote authentication,
+                    the peer_id must be derived from the identity key.",
```

Comment:
Nit: error message spaceing.

Reply:
The readme is super out of date, I cannot touch that in this PR 🐑